wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,60 @@
+# Restricted Mode
+
+A restriction is a timeout from the community forced upon accounts that exhibit abnormal, suspicious, or rule-breaking behaviour.
+
+When an account is restricted, it is unable to interact with the community until the restriction is lifted. Restricted players can still continue playing and submitting scores, but their profile will be invisible to others.
+
+The following features are disabled for restricted accounts:
+- in-game chat
+- private messaging on the forums
+- posting on the forums
+- updating maps or profile details
+- store purchases
+
+## My account has been restricted! What do I do?
+
+If your account has been restricted, do not panic. Please follow these steps:
+- **Wait 24 hours.** Some restrictions are the results of automated processes. As we check every new restriction daily, please wait 24 hours - if your restriction was erroneous, it will be lifted within that time.
+- **Contact our support team.** If your account remains restricted after that time, please send an email to [accounts@ppy.sh](mailto:accounts@ppy.sh) **from the email address attached to your account** to ask for details on your restriction. Make sure you include your username, too!
+
+If your account has become restricted for breaking the [rules](/wiki/rules), a strict cooldown of three months will be enforced before you are allowed to send an appeal.
+
+If your original account is not restricted, you may log into it and continue playing normally. but make sure not to log into your multi-accounts from then on, as doing so may lead to further punishment.
```

Comment:
This feels very out of place. Suddenly we are talking about multi-accounts with no surrounding context.

Reply:
Should I remove that part or add a whole section on multi-accounting?